Added Mobile Support, Made Tab Buttons frame scrollable for more Tabs and alot more.

mstudio45 opened the pull request:
```diff
+ Added Mobile Support
+ Now using ScrollingFrame instead of Frame for more Tabs.
+ Added `ClearTextOnFocus` for Inputs.
+ Added `Delete Config` button to Save Manager.
+ Added `ShowCustomCursor`. If its set to false then the custom cursor will not show.
+ Added Video Backgrounds.
+ Added `Toggle UI` and `Lock  UI` buttons for Mobile users.
+ Added Delete Theme button to themes.
+ Added Instance Notifications. (Notification will go away after the Instance is deleted)
+ Added sound to Notifications. Example: Library:Notify(Text, Time/Instance, SoundId) or Library:Notify(Text, Time/Instance)
+ Fixed "Unknown" keybinds being triggered by one random key.

+ And I think thats all. You can fix some bugs but I had no issues with the Mobile Support and the new changes.
```

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update README.md

mstudio45 committed:
Update Example.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Example.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

notpoiu committed:
Added Instance Notification things

mstudio45 committed:
Merge pull request #2 from notpoiu/main

Added Instance Notification things

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update SaveManager.lua

notpoiu commented:
gotta love how @violin-suzutsuki ghosted you on this pull reqeust

notpoiu commented:
also theres one bug where it dosent allow you to scroll down any dropdowns in mobile, you can fix that
![image](https://github.com/violin-suzutsuki/LinoriaLib/assets/75510171/859c36cd-1481-4193-b362-f687ed28e837)

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update Library.lua

RectangularObject committed:
update Example.lua

RectangularObject committed:
cloneref

RectangularObject committed:
add missing getgenv().Linoria

RectangularObject committed:
fix toggle input sink

RectangularObject committed:
fix indent
thanks vsc.

RectangularObject committed:
whoops, wrong frame

RectangularObject committed:
add types to services
returns autocomplete functionality to vsc

RectangularObject committed:
fix highlight

RectangularObject committed:
actually fix highlight

RectangularObject committed:
fix highlight not updating
if the mouse was on a toggle and moved into a colorpicker, the highlight wouldn't update
now it should

RectangularObject committed:
fix highlight not updating?
"now it should," he said

RectangularObject committed:
fix colorpicker offset

RectangularObject committed:
improve custom cursor

RectangularObject committed:
fix strange error?

RectangularObject committed:
strange error fixed

RectangularObject committed:
optimize mouse

RectangularObject committed:
Revert "optimize mouse"

This reverts commit c653eac0b229820d5a6d54f3eff885996a15e667.

RectangularObject committed:
optimize mouse

RectangularObject committed:
Revert "fix colorpicker offset"

This reverts commit 7b54a64922cd9c4b42c74d2df9ac0cead59100e3.

RectangularObject committed:
fix togglelabel offset
also fixes the colorpicker offset

RectangularObject committed:
don't play sound if soundid is nil

RectangularObject committed:
add dropdown support for depbox

RectangularObject committed:
update Example.lua

RectangularObject committed:
restore original mouse icon state

RectangularObject committed:
add more colorpickers to example

RectangularObject committed:
fix indent

RectangularObject committed:
fix indent

RectangularObject committed:
Merge branch 'main' of https://github.com/RectangularObject/LinoriaLib

RectangularObject committed:
fix tab padding

RectangularObject committed:
close dropdowns when menu is closed

mstudio45 committed:
Merge pull request #3 from RectangularObject/main

Several fixes and additions

mstudio45 committed:
Update Example.lua

mstudio45 committed:
Update README.md

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
made Keybinds frame draggable

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

deividsleepy committed:
Update Library.lua

deividsleepy committed:
Update Library.lua

mstudio45 committed:
Merge pull request #5 from deividcomsono/main

feat: Visibility to Toggles and Sliders

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Update ThemeManager.lua

mstudio45 committed:
Create test.lua

mstudio45 committed:
Update test.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Delete test.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
feat: Added Tab:UpdateWarningBox

Tab:UpdateWarningBox({
    Visible = boolean, 
    Title = string, 
    Text = string
})

mstudio45 committed:
Update Library.lua

notpoiu committed:
refactor: remove ProcessButtonParams function

notpoiu committed:
debug: add debug statements to debug

notpoiu committed:
fix: fixed issue

notpoiu committed:
debug: add debug print statements

notpoiu committed:
fix: possible not loading issue with debug print statements

notpoiu committed:
debug: even more cursed oneline debugging

notpoiu committed:
refactor: remove debug statements, turns out we were just acoustic

mstudio45 committed:
Merge pull request #7 from notpoiu/main

refactor: refactored ButtonParamsCheck function

mstudio45 committed:
Revert "refactor: refactored ButtonParamsCheck function"

mstudio45 committed:
Merge pull request #8 from mstudio45/revert-7-main

Revert "refactor: refactored ButtonParamsCheck function"

notpoiu committed:
fix: how the flip did that get there

notpoiu committed:
Merge branch 'main' into main

mstudio45 committed:
Merge pull request #9 from notpoiu/main

refactor: ButtomParams stuff

notpoiu committed:
refactor: remove ButtonParamCheck

mstudio45 committed:
Merge pull request #10 from notpoiu/main

refactor: remove ButtonParamCheck

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
feat: check extended description

- Added Library.ShowToggleFrameInKeybinds
- Fixed mobile buttons not being draggable
- Fixed Draggable functions
- Fixed ColorPickers, ContextMenus and ModePickers not closing during Library:Toggle

mstudio45 committed:
Update Example.lua

mstudio45 committed:
Update Example.lua

mstudio45 committed:
Update Example.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Update Library.lua

mstudio45 committed:
feat: fixed ShowToggleFrameInKeybinds

mstudio45 committed:
fix: remove property

mstudio45 committed:
Update Library.lua

mstudio45 committed:
fix: update wrong resizing for UpdateWarningBox's frame

deividsleepy committed:
feat: insert OnUnload to table

mstudio45 committed:
Merge pull request #11 from deividcomsono/patch-1

feat: insert OnUnload to table

deividsleepy committed:
fix: ShowCustomCursor update by toggle

mstudio45 committed:
Merge pull request #12 from deividcomsono/patch-2

fix: ShowCustomCursor update by toggle

mstudio45 committed:
Update README.md

mstudio45 committed:
Update README.md

deividsleepy committed:
fix: GetState check GetFocusedTextBox

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Update Example.lua

deividsleepy committed:
feat: sub folders

rbxlscripts committed:
fix: subfolder not found (better then upio check)

rbxlscripts committed:
fix: line 248

deividsleepy committed:
fix: subfolder not found (better then upio check)

ding dong eat it up

deividsleepy committed:
style: whitespaces fuck

rbxlscripts committed:
fix: some bugs in savemanager, old variables, optimized some parts of code, update thememanager to match savemanager code style, added theme notifications, added Library.Options/Library.Toggles, updated example.lua

rbxlscripts committed:
fix: split path to parts for some executors

deividsleepy committed:
perf: dont set value if value is the same

deividsleepy committed:
style: stylished

deividsleepy committed:
fix: typo

notpoiu committed:
feat: add rich text support to label

deividsleepy committed:
feat: add rich text support to label

rbxlscripts committed:
fix: for whatever reason it wasnt checking if type exists and parser exists for toggles

deividsleepy committed:
feat: visible arg for button

deividsleepy committed:
style: add comma

mstudio45 committed:
Update SaveManager.lua

mstudio45 committed:
Merge pull request #13 from deividcomsono/main

refactor: factory has been busy factoring factors when no refactors

mstudio45 committed:
fix: load auto load config should be runned by the script not the savemanager

deividsleepy committed:
feat: visibility for dropdown

deividsleepy committed:
fix: dropdown text not hiding

mstudio45 committed:
deivid stop

feat: added Visible to Dropdowns

notpoiu committed:
feat: add option to expose label to options table

mstudio45 committed:
Merge pull request #15 from notpoiu/main

feat: add option to expose label to options table

notpoiu committed:
feat: label creation table support

ex:
```lua
Groupbox:AddLabel("LabelIdx", {
    Text = "Cool",
    DoesWrap = true
})
```

notpoiu committed:
docs: add documentation

notpoiu committed:
fix: missing argument #1

notpoiu committed:
fix: erm

notpoiu committed:
fix: all errors

notpoiu committed:
fix: ab

notpoiu committed:
fix: assert

mstudio45 committed:
Merge pull request #16 from notpoiu/main

feat: label creation table support

mstudio45 committed:
feat: SetText, OriginalText, Text

mstudio45 committed:
feat: Tab:SetName, OriginalName, Name; Window = OriginalTitle, Title

mstudio45 committed:
fix: wrong var

mstudio45 committed:
feat: Labels, Buttons

deividsleepy committed:
fix: dropdown scroll not scrolling for mobile

mstudio45 committed:
Merge pull request #17 from deividcomsono/main

Fix Dropdown ScrollingFrame not working due to Frame event

deividsleepy committed:
fix: key-value dropdown not showing

mstudio45 committed:
Merge pull request #18 from deividcomsono/main

fix: key-value dropdown not showing

mstudio45 committed:
feat: fix for thread cannot access error, display runs after updating

mstudio45 committed:
fix: revert

mstudio45 committed:
feat: disabled option for toggle

mstudio45 committed:
fix

mstudio45 committed:
feat: Disabled for every UI element (too lazy to do Input), fix: tooltip disabled not set on creation

mstudio45 committed:
feat: improve example.lua, fix DisabledTooltip

mstudio45 committed:
fix: subbutton not checking the correct tooltip

mstudio45 committed:
perf: disconnect connections on tooltip deletion

mstudio45 committed:
perf: recode ApplyBackgroundVideo, remove unused variables

mstudio45 committed:
fix: yes

notpoiu committed:
feat: Disabled Values in Dropdowns

notpoiu committed:
docs: disabled values

notpoiu committed:
fix: Indentation fix

mstudio45 committed:
Merge pull request #19 from notpoiu/main

feat: Add Disabled Values to Dropdowns

mstudio45 committed:
feat: dont run callback if you run SetValue with the current value

mstudio45 committed:
fix: fix

notpoiu commented:
why has violin not merged this yet

mstudio45 commented:
234 commits 🤔

notpoiu commented:
no merge conflicts, LGTM!! 😱

mstudio45 committed:
Update README.md

mstudio45 committed:
Update README.md

mstudio45 committed:
feat: fix DisabledTooltip

mstudio45 committed:
feat: actually fix DisabledTooltip

mstudio45 committed:
Update README.md

mstudio45 committed:
fix: DisabledValues (what the flip upio)

mstudio45 committed:
indentation

mstudio45 committed:
fix: ApplyBackgroundVideo

mstudio45 committed:
fix: oops

asleepa committed:
added toggling boolean to library:toggle

mstudio45 committed:
Merge pull request #20 from asleepa/main

feat: added optional toggling boolean to library:toggle

mstudio45 committed:
feat: type check

mstudio45 committed:
fix: simple empty string check for tooltips

mstudio45 committed:
feat: Slider.Prefix/Suffix, Slider:SetPrefix/SetSuffix, Dropdown.Searchable

mstudio45 committed:
feat: readme

mstudio45 committed:
Update README.md

mstudio45 committed:
feat: new update

mstudio45 committed:
feat: use SafeCallback for OnChanged

mstudio45 committed:
feat: another thing

mstudio45 committed:
fix: changelog typo

mstudio45 committed:
feat: the flip man

mstudio45 committed:
feat: improve Config.Size

mstudio45 committed:
feat: tostring for notify

mstudio45 committed:
fix: dropdown item list overflow

mstudio45 committed:
fix: GetPlayersString(true)

mstudio45 committed:
fix: small ui (hopefully)

mstudio45 committed:
feat: make SafeCallback throw error instead of warn

mstudio45 committed:
fix: switch to if then variables

mstudio45 committed:
feat: new update

mstudio45 committed:
feat: another update

mstudio45 committed:
feat: richtext on notify

deividsleepy committed:
Options > Labels

mstudio45 committed:
Merge pull request #21 from deividcomsono/patch-3

Options > Labels

Blukery committed:
feat: DPI Scale

mstudio45 committed:
Update Library.lua

notpoiu committed:
refactor: made code follow linorialib repo conventions

Blukery committed:
Merge pull request #1 from notpoiu/main

refactor: made code follow linorialib repo conventions

mstudio45 committed:
Update Library.lua

mstudio45 committed:
Merge pull request #22 from Blukery/main

feat: DPI Scale

deividsleepy committed:
perf: content

deividsleepy committed:
fix: real

mstudio45 committed:
Merge pull request #24 from deividcomsono/patch-5

erm what the sigma

notpoiu commented on Library.lua:
🔥🔥🙏
